eventlet/greenio/base.py

```diff
@@ -431,6 +431,13 @@ def settimeout(self, howlong):
     def gettimeout(self):
         return self._timeout
 
+    if six.PY3:
```

Comment:
You don't mind that the std library in Python 2 doesn't have this? Happy to make it available to Python 2 as well, I just thought it better to try to be compatible with the standard library.